src/System.Console/src/System/ConsolePal.Windows.cs

```diff
@@ -218,6 +329,12 @@ private static ConsoleColor ColorAttributeToConsoleColor(Interop.mincore.Color c
             return (ConsoleColor)c;
         }
 
+        private static Interop.mincore.CONSOLE_SCREEN_BUFFER_INFO GetBufferInfo()
+        {
+            bool unused;
+            return GetBufferInfo(true, out unused);
+        }
+
```

Comment:
I have added some tests for Unix. I will add for Windows once IsXXXRedirected is checked in. It will make writing tests easier :)